tests/unit/e_db_mysqlTest.php

```diff
@@ -286,17 +286,18 @@ public function testUpdate()
 			$db->delete('tmp');
 
 			// Test 1
-			$expected = $db->update('tmp', array('tmp_ip' => '127.0.0.1', 'tmp_time' => time(), 'tmp_info' => 'test 1', 'WHERE' => 'tmp_ip="127.0.0.1"'));
+			$expected = $db->update('tmp', array('tmp_ip' => '127.0.0.1', 'tmp_time' => time(), 'tmp_info' => 'Update test 1', 'WHERE' => 'tmp_ip="127.0.0.1"'));
 			$this->assertEmpty($expected, "Test 1 update() failed (not empty {$expected})");
```

Comment:
This test could be improved by verifying that `tmp_info` was changed to `"Update test 1"` in `tmp_ip="127.0.0.1"` by querying the value before, asserting that it matches the previous value, then querying the value after and asserting that it matches the new value.